.github/workflows/assemble-turnstile-data.yml

```diff
@@ -0,0 +1,61 @@
+# Assembles Turnstile ingestion package (reusable workflow)
+
+on:
+  workflow_call:
```

Comment:
I don't think `workflow_call` should have a trailing colon (https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_call)

Reply:
Yes, you're correct.  I'd copied this file from build-virtual-machine.yml where the `workflow_call` had been followed by an input, and I'd deleted the input but not the trailing colon.  I'll go ahead, merge the pull request, and if CI fails immediately, I'll fix it, otherwise I'll wait for CI to find any other problems that I can fix at the same time too.